src/components/containers/seo.js

```diff
@@ -146,7 +146,19 @@ const SEO = ({ description, meta, title, no_index }) => {
                       ]
                     : []),
             ].concat(meta)}
-        ></Helmet>
+        >
+            <script
```

Comment:
so we will still load datadog js in localhost?

Reply:
@mustofa-binary loading the the script yes, but it should only run in production from the `window.location.hostname === 'www.deriv.com' &&...` check

Reply:
is it possible to defer?

Reply:
I think we don't want to defer since data dog will measure load time start and end times

Reply:
ah, i see, alright then 👍